src/devices/vesternet.ts

```diff
@@ -200,6 +200,55 @@ const definitions: Definition[] = [
             await reporting.batteryPercentageRemaining(device.getEndpoint(1));
         },
     },
+    {
+        fingerprint: [{modelID: 'DoorSensor-ZB3.0', softwareBuildID: '2.01'}, {modelID: 'DoorSensor-ZB3.0', softwareBuildID: '2.1'}, {modelID: 'DoorSensor-ZB3.0', softwareBuildID: '2.03'}, {modelID: 'DoorSensor-ZB3.0', softwareBuildID: '2.04'}],
```

Comment:
Do we really need to specify all `softwareBuildID`? I prefer to have a single `{modelID: 'DoorSensor-ZB3.0', manufacturerName: 'MANUFACTURER_NAME_FROM_DATABASE_DB'}`

Reply:
Similar to other Vesternet devices, these devices are rebranded from the underlying hardware manufacturer, so I can only validate and test against firmware versions that have been supplied under the Vesternet branding.  

They might be different to those supplied to other companies that also rebrand them, so I would prefer to only submit code that is matched explicitly to the versions that Vesternet sell.

(I know for example that firmware version 2.02 in the contact sensor had issues with the IAS Cluster and didn't work correctly, hence I've excluded that version).